Contained app/Contained app/SettingsViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  SettingsViewController.swift
+//  Contained app
+//
+//  Created by Drew Miller on 5/18/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController {
+
+    @IBAction func rollToggle(_ sender: UISwitch) {
+    sender.isSelected.toggle()
```

Comment:
Take the value of the switch and assign that value to an instance variable for our class object.